examples/containers/multi/helpers/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Change 
+ARG couchbase_server_image=couchbase/server:enterprise-6.6.3
```

Comment:
This should be uppercase, I think this was a change that @markspolakovs did somewhere else. We want to pass in from the makefile the standard variables.